spec.html

```diff
@@ -4917,9 +4918,9 @@ <h1>ToUint32 ( _argument_ )</h1>
       <p>The abstract operation ToUint32 takes argument _argument_. It converts _argument_ to one of 2<sup>32</sup> integer values in the range 0 through <emu-eqn>2<sup>32</sup> - 1</emu-eqn>, inclusive. It performs the following steps when called:</p>
       <emu-alg>
         1. Let _number_ be ? ToNumber(_argument_).
-        1. If _number_ is *NaN*, *+0*, *-0*, *+&infin;*, or *-&infin;*, return *+0*.
+        1. If _number_ is *NaN*, *+0*<sub>𝔽</sub>, *-0*<sub>𝔽</sub>, *+&infin;*, or *-&infin;*, return *+0*<sub>𝔽</sub>.
         1. Let _int_ be the Number value that is the same sign as _number_ and whose magnitude is floor(abs(_number_)).
```

Comment:
We do some weird gymnastics by converting this to a Number before we apply the modulo. Let's convert to Number only at the end.

Reply:
are we sure the web doesn't depend on some of these gymnastics? iow, does the current logic introduce any observable weirdness that needs to be preserved?

Reply:
This is working with small integers, so no.